src/components/photoIndexIndicator/PhotoIndexIndicator.tsx

```diff
@@ -0,0 +1,24 @@
+import { PhotoIndexIndicatorAbove5, renderPhotoIndexIndicator } from './helper';
```

Comment:
rename to 'helpers'

there is more than one helper method, we can even see so in this import